modules/_labs/drawer/react/stories/stories.tsx

```diff
@@ -64,7 +64,7 @@ storiesOf('Labs|Drawer/React', module)
         <Drawer
           header={
             <DrawerHeader
-              closeIconLabel={'Close'}
+              aria-label={'Close'}
```

Comment:
Shouldn't these be `closeIconAriaLabel` now? I would only expect to use `aria-label` if it were being applied to the header container element itself.